### DISCUSSION TITLE
Ban history (/history)

### DISCUSSION THREAD

**Discussion by BlakeTnr** [EssentialsX/Essentials#4105] [Ideas and Feature Suggestions] [answered]
Similar to the Litebans "/history" command. It would allow you to see the users ban history.

Could this take up a lot of space?
Kinda, I feel like if you just do some text compression then that issue would be mitigated.

Is it worth the space?
I think this feature is essential to anyone running a server that has staff. With this they could see previous punishments a user has or previous punishments a staff member has given. This could useful be for appeals, staff management, or just in general keeping history which normally is very helpful.

Shouldn't we just let Litebans take care of this?
In my case I know I use Litebans just because they have a history system. Minecraft already has a built in ban system that was already made better by EssentialsX. I think that for a full solution for bans and punishments then a punishment history is necessary. This could help the ban system be more functional and lessen the need for another punishment plugin.

Couldn't this effect current servers upgrading?
There is probably a way to just make it hook into current bans from bukkit and convert them. Or keep them and use some sort of linking system. Converting is probably better cause if you use compression hen it would lessen the space.

**Comment by JRoy** [comment, marked as answer]
EssentialsX just use's vanillas ban system and nothing more. We do not plan on storing ban histories at this time.
